### ISSUE TITLE
task input_method 'powershell' fails validation

### ISSUE THREAD

**Issue by jessereynolds** [puppetlabs/pdk#369]
If you have a task with input_method of 'powershell' pdk fails validation, eg:

```json
{
  "description": "Does something",
  "input_method": "powershell",
  "parameters": ...snip...
```

`pdk validate` 

```
error: task-metadata-lint: tasks/foo.json: The property '#/input_method' value "powershell" did not match one of the following values: stdin, environment
```

bolt docs describe the 'powershell' input_method: https://puppet.com/docs/bolt/0.x/writing_tasks.html#task-metadata-reference

**Comment by jessereynolds** [comment]
I've created [a pull request](https://github.com/puppetlabs/puppet-forge-api/pull/400) against puppet-forge-api to fix the `task.json` task json schema file served by the forge.

**Comment by DavidS** [comment]
This should have been shipped with pdk 1.3.

Feel free to open another report, if there is still problems with this (or anything else). Thanks for your time & work!
